package/Linux/gateway/debian/postinst

```diff
@@ -3,14 +3,18 @@
 # System-wide package configuration.
 DEFAULTS_FILE="/etc/default/devolutions-gateway"
 
+if [ ! -f $DEFAULTS_FILE ]; then
+	/bin/touch $DEFAULTS_FILE
+fi
```

Comment:
You mention that too in your original post, but I’m not sure we want to start creating this file at all given that we do not provide any option for configuring the service. We’re probably better off deleting the line above declaring `DEFAULTS_FILE`.